src/features/defi/providers/foxy/components/FoxyManager/Overview/Claim/ClaimRoutes.tsx

```diff
@@ -38,7 +38,7 @@ export const ClaimRoutes: React.FC<ClaimRouteProps> = ({ onBack, accountId }) =>
     assetNamespace,
     assetReference,
   })
-  const { data: foxyBalancesData } = useFoxyBalances()
+  const { data: foxyBalancesData } = useFoxyBalances({ accountNumber: 0 })
```

Comment:
This is wrong, we're in the context of a modal route and should be able to get the data for the right accountNumber / accountId

Reply:
7c2a92e33e22eacbf6eb5e95dfe2f63154f128e7